source/common/config/well_known_names.cc

```diff
@@ -70,12 +70,33 @@ TagNameValues::TagNameValues() {
   // http.[<stat_prefix>.]fault.(<downstream_cluster>.)<base_stat>
   addRegex(FAULT_DOWNSTREAM_CLUSTER, R"(^http(?=\.).*?\.fault\.((.*?)\.)\w+?$)", ".fault.");
 
-  // listener.[<address>.]ssl.cipher.(<cipher>)
-  addRegex(SSL_CIPHER, R"(^listener(?=\.).*?\.ssl\.cipher(\.(.*?))$)");
+  // listener.[<address>.]ssl.ciphers.(<cipher>)
+  addRegex(SSL_CIPHER, R"(^listener(?=\.).*?\.ssl\.ciphers(\.(.*?))$)", ".ssl.ciphers.");
```

Comment:
This probably should be `SSL_CIPHER_SUITE` to match the cluster version.

Reply:
Removed `SSL_CIPHER` tag completely in favour of `SSL_CIPHER_SUITE`